mono/mini/aot-compiler.c

```diff
@@ -7950,6 +7951,8 @@ mono_aot_parse_options (const char *aot_options, MonoAotOptions *opts)
 			opts->no_opt = TRUE;
 		} else if (str_begins_with (arg, "clangxx=")) {
 			opts->clangxx = g_strdup (arg + strlen ("clangxx="));
+		} else if (str_begins_with (arg, "depfile=")) {
```

Comment:
@vargaz could you please document this (including the expected format)

/cc @lambdageek